setup.py

```diff
@@ -51,7 +51,7 @@
 one_compiler_per_distribution = pl_version == ">=0.32,<=0.34"
 if one_compiler_per_distribution:
     entry_points = {
-        "pennylane.plugins": "cudaq = catalystcuda:CudaQDevice",
+        "pennylane.plugins": "softwareq.qpp = catalyst.cuda:CudaQDevice",
```

Comment:
do we also need to register the `nvidia.cutensornet` and `nvidia.custatevec` devices?

Reply:
I haven't tested them so I don't want to add them yet. I think that can easily be another PR.